app/assets/stylesheets/urls.scss

```diff
@@ -4,6 +4,23 @@ p {
   margin-top: 70px;
 }
 
-.urls-index-header {
-  padding-top: 80px;
+.index-header{
+  margin-top: 50px;
+  margin-left: 30px;
+}
+
+.search-icons {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.